src/app/api/gql/fragment-dataset-overview.graphql

```diff
@@ -3,6 +3,53 @@ fragment DatasetOverview on Dataset {
   createdAt
   lastUpdatedAt
   metadata {
+     currentUpstreamDependencies {
```

Comment:
As we've inlined lineage query here, we don't need old lineage to exist.
Same for metadata's "currentTrasnform".